src/main/kotlin/no/nav/amt/aktivitetskort/service/AktivitetskortService.kt

```diff
@@ -193,13 +197,32 @@ class AktivitetskortService(
 		endretAv = EndretAv("amt-aktivitetskort-publisher", IdentType.SYSTEM),
 		endretTidspunkt = LocalDateTime.now(),
 		avtaltMedNav = deltaker.status.type !in IKKE_AVTALT_MED_NAV_STATUSER,
-		oppgave = null,
+		oppgave = oppgaver(deltaker, deltakerliste),
 		handlinger = getHandlinger(deltaker, deltakerliste.tiltak.type),
 		detaljer = Aktivitetskort.lagDetaljer(deltaker, deltakerliste, arrangor),
 		etiketter = listOfNotNull(deltakerStatusTilEtikett(deltaker.status)),
 		tiltakstype = deltakerliste.tiltak.type,
 	)
 
+	private fun oppgaver(deltaker: Deltaker, deltakerliste: Deltakerliste): OppgaveWrapper? {
+		if (!unleashToggle.erKometMasterForTiltakstype(deltakerliste.tiltak.type)) {
+			return null
+		}
+		if (deltaker.status.type != DeltakerStatus.Type.UTKAST_TIL_PAMELDING) {
+			return null
+		}
+
+		return OppgaveWrapper(
+			ekstern = Oppgave(
+				tekst = "Du har mottatt et utkast til påmelding",
+				subtekst = "Før vi sender dette til Muligheter AS vil vi gjerne at du leser gjennom. Hvis du godkjenner utkastet blir du meldt på, vedtaket fattes og Muligheter AS mottar informasjon.",
```

Comment:
Vi bør kanskje ikke hardkode inn arrangørnavn her? 😅

Reply:
Ville teste det litt for fort 😅